### DISCUSSION TITLE
interface

### DISCUSSION THREAD

**Discussion by maximL2002** [Orama-Interactive/Pixelorama#892] [Q&A]
![image](https://github.com/Orama-Interactive/Pixelorama/assets/141019405/aed3ed11-98f8-4b65-adda-7cfa39b8db63)
Hello, does anyone know how can I fix this?

**Comment by dphaldes** [comment]
Looks like a font issue. Can you provide some details about platform and how you installed the software?

**Reply by maximL2002** [reply]
I'm using linux mint cinnamon 21.2
and I have istall it from software manager

**Reply by dphaldes** [reply]
Software manager? Is it a flatpak ?

**Reply by maximL2002** [reply]
yes

**Reply by maximL2002** [reply]
I also tried to install it from itch.o and have the same issue
